### ISSUE TITLE
Required for nested parsers

### ISSUE THREAD

**Issue by stasbel** [omni-us/jsonargparse#43]
This does work:

```python3
from jsonargparse import ArgumentParser, ActionParser
p1 = ArgumentParser()
p1.add_argument('--op1', required=True)

>>> p1.parse_args(['--op1=1'])
```

But this doesn't work:

```python3
from jsonargparse import ArgumentParser, ActionParser
p1 = ArgumentParser()
p1.add_argument('--op1', required=True)

p2 = ArgumentParser()
p2.add_argument('--op2', action=ActionParser(parser=p1))

>>> p2.parse_args(['--op2.op1=1'])
# error: Configuration check failed :: Key "op1" is required but not included in config object or its value is None.
```

**Comment by mauvilsa** [comment]
Thank you for reporting the bug! This was a simple omission which only required a simple fix. It is now fixed in the current version in master.

**Comment by mauvilsa** [comment]
This is now fixed and part of release v3.7.0.
